index.html

```diff
@@ -875,6 +875,15 @@ <h2 id='ldpcvpost'>HTTP POST</h2>
           Location Discovery</a>.
         </p>
       </section>
+      <section id="append-mode">
+        <h2>Append Mode</h2>
+        <p>
+          The [[!SOLIDWEBAC]] <a href="https://github.com/solid/web-access-control-spec#modes-of-access">Modes of
+          Access</a> section mentions the <code>acl:mode</code> value <code>acl:Append</code>.  In the context of a
+          Fedora implementation, <code>acl:Append</code> should be understood as operations that only append, such as
```

Comment:
I agree with the sentiment of this PR. Would it be helpful to add MUST/SHOULD language so that this text is more directly testable?

Reply:
@awoods I think some testable outcomes of this are, when a client makes a request whose effective ACL allows the authenticated user to `acl:Append`:
* POST MUST be allowed
* PATCH that deletes triples MUST be denied
* PATCH that only adds triples SHOULD be allowed (SHOULD here to allow implementations the option of not checking and just denying all PATCHes)
* PUT MUST be denied
* DELETE MUST be denied

Is there anything missing from this list?

Reply:
That sounds perfect, @escowles . Would you be interested in adding the clarity of your bullets in the text of the PR?

Reply:
@escowles - seems like there are different cases mixed up here depending on the resource type affected:

LDP-NR:
  * POST, PATCH, PUT, DELETE MUST all be rejected

LDP-RS:
  * PATCH that deletes triples MUST be denied
  * PATCH that only adds triples SHOULD be allowed (SHOULD here to allow implementations the option of not checking and just denying all PATCHes)
  * PUT to replace the LDP-RS MUST be denied
  * DELETE MUST be denied

LDPC has LDP-RS rules above plus:
  * POST of a new resource in the LDPC MUST be allowed
  * PUT of a new resource in the LDPC MAY be allowed (per https://fcrepo.github.io/fcrepo-specification/#httpPUTcreate )

Reply:
@zimeon I was just drafting the language and starting to think about the implications of different resource types, so thanks for laying it out so clearly!

Reply:
I'd disagree about LDP-NR "POST, PATCH, PUT, DELETE MUST all be rejected".  It's possible that an implementation may be able to implement PATCH in a meaningful way for certain media types, which presumably leads to the possibility of appending to it.  The language in the spec ought to make appending to binaries allowable.

Reply:
I agree with @birkland's comment and that seems handled by @escowles new changes